auction-server/src/opportunity/repository/models.rs

```diff
@@ -89,6 +90,7 @@ pub struct OpportunityMetadataSvmProgramSwap {
     #[serde(default = "default_cancellable")]
     pub cancellable:                          bool,
     pub minimum_lifetime:                     Option<u32>,
+    pub other_quotes:                         Vec<OtherQuote>,
```

Comment:
It doesn't feel great to have such a big struct here. 
Could we alternatively just return the searcher token amount?